examples/match/match.bal

```diff
@@ -17,13 +17,14 @@ function getValue(string key) returns string?|KeyNotFoundError {
 
 // This function prints a custom message depending on the type of the result.
 function print(string?|KeyNotFoundError result) {
-    // This match statement executes the code block based on the type of the result variable reference.
-    match result {
-        string value => io:println("value: " + value);
-        () => io:println("value is ()");
-        KeyNotFoundError e => {
-            io:println(e.message);
-        }
+    // This type-guard check (in other word, this is `is` check) which checks the type of the reference variable. Inside
```

Comment:
Change to "This is a type-guard check (also known as `is` check) that checks the type of the reference variable. Inside the `if` block, the reference variable is cast to the respective `is` checked type."